src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -429,6 +431,21 @@ bool TryDeleteRange(out DateTime? next)
                         //     return true;
                         // }
 
+                        if (canDeleteEntireSegment && readOnlySegment.NumberOfLiveEntries > 1)
```

Comment:
What about the commented code above? This is now no longer relevant, no? Should remove it.